ui/src/app/routing/guards.tsx

```diff
@@ -29,19 +29,19 @@ export const signInGuard: Guard = {
   redirectPath: '/login',
 };
 
-const userIsEnabled = (userDisabledInDb: boolean) =>
-  !userDisabledInDb &&
-  eligibleForTier(profileStore.get().profile, AccessTierShortNames.Registered);
+export const userIsDisabled = (userDisabledInDb: boolean) =>
+  userDisabledInDb ||
+  !eligibleForTier(profileStore.get().profile, AccessTierShortNames.Registered);
```

Comment:
More out of curiosity, when would a user not be eligible for the register tier? Does that just mean that they haven't done training for RT, or are they simply not allowed?

Reply:
User need to complete all of the following in order to gain RT access:

- Turn on Google 2-Step Verification
- Verify your identity with Login.gov 
- Connect your eRA Commons account
- Complete All of Us research Registered Tier training

Reply:
The above comment is incorrect, the eligibility is decided by user's institution.